util/ConfigFileProcess.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _ConfigFileProcess_h_
+#define _ConfigFileProcess_h_
+
+#include <vector>
+#include <string>
+#include <map>
```

Comment:
Almost none of those headers is needed by the `ConfigFileProcess` declaration so don't pollute this header with them.